guides/common/modules/proc_creating-a-lifecycle-environment-path.adoc

```diff
@@ -18,17 +18,16 @@ Then optionally add additional environments to the environment paths.
 [options="nowrap" subs="+quotes"]
 ----
 $ hammer lifecycle-environment create \
---name "_Environment Path Name_" \
+--name "_My_Lifecycle_Environment_Path_Name_" \
 --description "_Environment Path Description_" \
---prior "Library" \
```

Comment:
this seems unintentional to me.

Reply:
In the ticket, the reporter said it should not be necessary to include ' --prior "Library" ' saying that "The command knows it has to create a new Environment Path simply from the fact that the one referred to does not exist, so this option seems to be redundant."

What are your thoughts @maximiliankolb?

Reply:
I could not reproduce this on Foreman 3.11 yet:

````
# hammer lifecycle-environment create --name "test-me" --organization-id 1
Could not create environment:
  Error: At least one of options --prior, --prior-id is required.
  
  See: 'hammer lifecycle-environment create --help'.
# hammer lifecycle-environment create --name "test-me-2" --organization-id 1 --prior "Library"
Environment created.
````

That makes me belive that it's possibly a new(er) feature; so you could not cherry-pick this to 3.11 and below. Does it work for you on Foreman 3.13/3.12?